.github/actions/upload_android_app/action.yml

```diff
@@ -45,8 +41,8 @@ runs:
     - name: Set up JDK
       uses: actions/setup-java@v4
       with:
-        distribution: 'temurin'
-        java-version: '21'
+        distribution: "temurin"
+        java-version: "17"
```

Comment:
Too new at 21, so lowered to 17!
- related: https://github.com/Favosia/favosia_flutter/pull/2444